xblockutils/base_test.py

```diff
@@ -80,7 +80,10 @@ def go_to_page(self, page_name, css_selector=None):
             css_selector = self._default_css_selector
 
         self.browser.get(self.live_server_url)
-        self.browser.find_element_by_link_text(page_name).click()
+        target_url = self.browser.find_element_by_link_text(page_name).get_attribute('href')
+        if view_name:
+            target_url += '%s/' % view_name
```

Comment:
It's used quite a bit. A quick search for the operator found hundreds of results. :)